inc/minishell_error.h

```diff
@@ -58,8 +58,12 @@ typedef enum e_error {
 	ERR_SIGNAL, ///< Setup of signal handler failed.
 	ERR_NO_INPUT, ///< Input is empty.
 	ERR_ABORT, ///< Abort pipeline.
-	ERR_HEREDOC_EOF ///< Heredoc stopped by Ctrl+D
+	ERR_HEREDOC_EOF, ///< Heredoc stopped by Ctrl+D
+	ERR_DIR, ///< Cmd is a directory.
+	ERR_NO_DIR ///< Directory of file not found
```

Comment:
Did you mean Directory OR file not found?

Reply:
yes, corrected